redhat-developer/developers.redhat.com
# DEVELOPER-5915 - Updated rich text assembly visual styles

### JIRA Issue Link
https://issues.jboss.org/browse/DEVELOPER-5915

### Verification Process

* Log in to the Drupal PR environment [here](https://developer-preview-3000.ext.us-west.dc.preprod.paas.redhat.com/user/login)
* Verify that this is no 'No background' visual style option on the Rich Text assembly [here](https://developer-preview-3000.ext.us-west.dc.preprod.paas.redhat.com/assembly/add/rich_text)
